pipeline.yml

```diff
@@ -91,10 +91,10 @@ jobs:
   plan:
   - aggregate:
     - get: broker-src
-      passed: [unit-test-uaa-guard-service]
+      passed: [unit-test-uaa-guard-service, push-uaa-guard-service-staging]
```

Comment:
Since the staging deploy already includes a `passed` constraint on the unit tests, can't we just constrain the production deploys to use the resources passed by the staging deploys?